### DISCUSSION TITLE
Command block limits

### DISCUSSION THREAD

**Discussion by Nomadicbrain** [PurpurMC/Purpur#1007] [__Dismissed__] [answered]
Please add a config that allows the admins to set the string limit for command blocks and not lock it at 2048 which is far below the mojang default limit. I reach far over this limit trying to give formatted lore books to players.

**Comment by sovos15** [comment]
Second, getting kicked as OP is big lame.

**Comment by katekerllenevich** [comment, marked as answer]
This limit appears to be client-sided, I cant find any references to `2048` in the code (or wiki) relating to command blocks. Please let me know if I'm wrong (I probably am).

**Comment by Nomadicbrain** [comment]
We did some testing and the change happens when either spigot or purpur is
installed. Does not occur on a vanilla world. We originally tried purpur
because the spigot devs added it themselves and said they wouldn't change
it.
